src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLanguageService`IVsLanguageDebugInfo.cs

```diff
@@ -33,12 +50,40 @@ public int GetProximityExpressions(IVsTextBuffer pBuffer, int iLine, int iCol, i
 
         public int ValidateBreakpointLocation(IVsTextBuffer pBuffer, int iLine, int iCol, TextSpan[] pCodeSpan)
         {
+            var textBuffer = _editorAdaptersFactory.GetDataBuffer(pBuffer);
+            if (textBuffer == null)
+            {
+                // Can't resolve the text buffer, let someone else deal with this breakpoint.
+                return VSConstants.E_NOTIMPL;
+            }
+
+            var snapshot = textBuffer.CurrentSnapshot;
+            if (!ValidBreakpointLocation(snapshot, iLine, iCol))
+            {
+                // The point disappeared between sessions. Do not allow a breakpoint here.
+                return VSConstants.E_FAIL;
```

Comment:
Is there a more specific errorcode we can use here?

Reply:
Nope, the E_FAIL tells the requester "don't put a breakpoint here"